docs/deploying-airbyte/on-oci-vm.md

```diff
@@ -30,31 +30,40 @@ ssh -i private-key-file opc@oci-private-instance-ip -p 2200
 
 ### Install Docker
 
+```text
 sudo yum update -y
 
 sudo yum install -y docker
 
 sudo service docker start
 
 sudo usermod -a -G docker $USER
+```
+
 
 ### Install Docker Compose
 
-sudo wget [https://github.com/docker/compose/releases/download/1.26.2/docker-compose-$\(uname](https://github.com/docker/compose/releases/download/1.26.2/docker-compose-$%28uname) -s\)-$\(uname -m\) -O /usr/local/bin/docker-compose
+```text
+sudo wget https://github.com/docker/compose/releases/download/1.26.2/docker-compose-$(uname -s)-$(uname -m) -O /usr/local/bin/docker-compose
 
 sudo chmod +x /usr/local/bin/docker-compose
 
 docker-compose --version
+```
+
 
 ### Install Airbyte
 
+```text
 mkdir airbyte && cd airbyte
 
-wget [https://raw.githubusercontent.com/airbytehq/airbyte/master/{.env,docker-compose.yaml}](https://raw.githubusercontent.com/airbytehq/airbyte/master/{.env,docker-compose.yaml})
+wget https://raw.githubusercontent.com/airbytehq/airbyte/master/{.env,docker-compose.yaml}
 
 which docker-compose
 
-sudo /usr/local/bin/docker-compose up -d
+docker-compose up -d
```

Comment:
Should this still be sudo? I think this is the only non-formatting change here.

Reply:
It will work without sudo. Added the text tags also to avoid errors on copying pasting

Reply:
👍